play-services-location/core/provider/src/main/kotlin/org/microg/gms/location/network/wifi/MovingWifiHelper.kt

```diff
@@ -403,6 +404,34 @@ class MovingWifiHelper(private val context: Context) {
             }
         }
 
+        private val SOURCE_EUROSTAR = object : MovingWifiLocationSource("https://www.ombord.info/api/jsonp/position/") {
```

Comment:
Please name it `SOURCE_OMBOARD`. `omboard.info` is owned by Icomera, they likely have this system deployed on other trains as well, so it's not specific to Eurostar.

Reply:
Done in https://github.com/microg/GmsCore/pull/2821/commits/0670dc72d386332e4f91d1eea8ddae28805f6faa - thanks!